rules/ml/ml_auth_rare_hour_for_a_user_to_logon.toml

```
@@ -0,0 +1,29 @@
+[metadata]
+creation_date = "2021/06/10"
+maturity = "production"
+updated_date = "2021/06/10"
+
+[rule]
+anomaly_threshold = 75
+author = ["Elastic"]
+description = """
+A machine learning job detected a user logging in at a time of day that is unusual for the user.
+This can be due to credentialed access via a compromised account when the user and the threat 
+actor are in different time zones. In addition, unauthorized user activity often takes place 
+during non-business hours.
+"""
+false_positives = [
+    """
+    """,
+]
+from = "now-30m"
+interval = "15m"
+license = "Elastic License"
```

All of these `license` should be `v2`

```suggestion
license = "Elastic License v2"
```